static/room.js

```diff
@@ -32,11 +32,11 @@ function chooseOrUnchooseCard(card) {
 }
 
 function displayHand(hand) {
-    var hand_to_render = '';
+    var handToRender = '';
```

Comment:
It's just the standard javascript style. Underscores are the prescribed python style.